.gitignore

```diff
@@ -237,3 +237,6 @@ tags
 [._]*.un~
 
 # End of https://www.gitignore.io/api/eclipse,vim,python,git
+
+# vscode
+.vscode/
```

Comment:
This should be done with `gitignoreio` instead. Update `gitignore.io` with entry "VisualStudioCode".